test/src/kbasesearchengine/test/search/ElasticIndexingStorageTest.java

```diff
@@ -19,11 +20,10 @@
 import java.util.function.Function;
 import java.util.stream.Collectors;
 
+import kbasesearchengine.search.*;
```

Comment:
Please don't use wildcard imports.

Reply:
ughh I think thats due to intellij auto formatting. need to figure out how to turn that off